shared/login/register/paper-key/index.js

```diff
@@ -23,12 +25,17 @@ class PaperKey extends Component {
 }
 
 PaperKey.propTypes = {
-  onSubmit: React.PropTypes.func.isRequired
+  onSubmit: React.PropTypes.func.isRequired,
+  onBack: React.PropTypes.func.isRequired
 }
 
 export default connect(
   state => state,
-  null,
+  dispatch => {
+    return {
+      onBack: () => dispatch(cancelLogin())
+    }
```

Comment:
:sheep: :sheep: this can be `dispatch => ({onBack: () => dispatch(cancelLogin)})`